pkg/gatherers/clusterconfig/const.go

```diff
@@ -30,7 +30,8 @@ var (
 	// logDefaultLimitBytes is used in calls to common.CollectLogsFromContainers
 	logDefaultLimitBytes = int64(1024 * 64) // maximum 64 kb of logs
 
-	defaultNamespaces           = []string{"default", "kube-system", "kube-public", "openshift"}
+	defaultNamespaces = []string{"default", "kube-system", "kube-public", "openshift"}
```

Comment:
@rluders If VMIs exist in other namespaces will they also be collected? Im asking, since the VMIs can be create for any namespace.

Reply:
Yes, there is no filter being applied to the resource. This line was changed by the `go fmt`, it doesn't affect the VMIs resource gathering.